MooTools-More-1.6.0.js

```diff
@@ -5014,8 +4722,6 @@ requires:
 
 provides: [URI]
 
-used: Various places
```

Comment:
I didn't bother adding these comments back cause the remaining ones aren't that useful, and it's an extra step every time we want to regenerate this file